apps/studio/src/types/schema.ts

```diff
@@ -4,14 +4,24 @@ import {
   type UISchemaElement,
   type VerticalLayout,
 } from '@jsonforms/core'
+import { type IsomerComplexComponentProps } from '@opengovsg/isomer-components'
 
-export type IsomerJsonSchema = JsonSchema & {
+export type IsomerExtendedJsonSchema = JsonSchema & {
   groups?: Array<{
```

Comment:
double checking my understanding here: JsonSchema is the schema definition as given by JsonForms lib. We are extending it to add a new key for `groups`

Reply:
Yup, codifying it here so that we are aware what are the things that were enhanced on top of the JSON Schema specifications.